docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '2.1'
+
+services:
+  zap-proxy:
+    image: owasp/zap2docker-weekly:latest
+    command: zap-x.sh -daemon -host 0.0.0.0 -config view.mode=attack -config scanner.attackOnStart=true -config api.disablekey=true -config api.addrs.addr.name=.* -config api.addrs.addr.regex=true -config connection.timeoutInSecs=120 -config database.recoverylog=false -config connection.proxyChain.hostName=reformmgmtproxyout.reform.hmcts.net -config connection.proxyChain.port=8080
```

Comment:
I had to configure `reformmgmtproxyout` here as zap `-config` entries.

Reply:
`proxyout.reform.hmcts.net` should be used

Reply:
why does it need to talk out btw?

Reply:
Payments